nengo_gui/components/netgraph.py

```diff
@@ -162,9 +162,41 @@ def _reload(self, code=None):
         orphan_components = []
         rebuild_components = []
 
+        # items that are shown in components, but not currently displayed
+        #  in the NetGraph (i.e. things that are inside collapsed
+        #  Networks, but whose values are being shown in a graph)
+        collapsed_items = []
+
         removed_items = list(removed_uids.values())
         for c in self.page.components[:]:
             for item in c.code_python_args(old_default_labels):
+                if item not in self.uids.keys() and item not in collapsed_items:
+                    # this Component depends on an item inside a collapsed
+                    #  Network, so we need to check if that component has
+                    #  changed or been removed
+                    try:
+                        old_obj = eval(item, old_locals)
+                    except:
+                        continue
```

Comment:
Why would evaluating this cause an exception? If I understood why, I would feel more comfortable with behaviour of continuing regardless of exceptions.

Also, if you do continue after an exception, isn't that going to cause an error in line Line 188 where you do `not isinstance(new_obj, old_obj.__class__)` since `old_obj` won't exist?